### ISSUE TITLE
is this project dead?

### ISSUE THREAD

**Issue by geddski** [tekezo/Karabiner#835]
I get the whole karabiner-elements thing, but I'm wondering if this project is ever going to work on macOS? Feels abandoned.

**Comment by dunkarooftop** [comment]
No, Karabiner-Element is replacement for Karabiner, one day Karabiner-Element will be just like the old Karabiner. but no one know how long it will take :(

**Comment by thorro** [comment]
In the meantine, I use https://ei-kana.appspot.com/ as it offers more functionality than Elements. I only miss per-app settings.
